meson.build

```
@@ -1,7 +1,7 @@
 project(
   'netpanzer',
   ['c', 'cpp'],
-  version: '0.9.0.07.555',
+  version: '0.9.0-rc8',
```

```suggestion
  version: '0.9.0-rc8-dev',
```

@winrid Technically this isn't the '0.9.0-rc8' version but a -dev or -git version or whatever you might prefer.

---

Yeah I think that's less confusing